src/main/kotlin/no/nav/familie/ef/søknad/service/EttersendingService.kt

```diff
@@ -4,24 +4,24 @@ import no.nav.familie.ef.søknad.api.dto.Kvittering
 import no.nav.familie.ef.søknad.integration.SøknadClient
 import no.nav.familie.ef.søknad.mapper.KvitteringMapper
 import no.nav.familie.ef.søknad.mapper.kontrakt.EttersendingMapper
-import no.nav.familie.kontrakter.ef.ettersending.EttersendingDto
+import no.nav.familie.kontrakter.ef.ettersending.EttersendelseDto
 import no.nav.familie.kontrakter.ef.ettersending.EttersendingResponseData
+import no.nav.familie.kontrakter.ef.felles.StønadType
 import org.springframework.stereotype.Service
 import java.time.LocalDateTime
 
 
 @Service
-class EttersendingService(private val søknadClient: SøknadClient,
-                          private val ettersendingMapper: EttersendingMapper) {
+class EttersendingService(private val søknadClient: SøknadClient) {
 
-    fun sendInn(ettersending: EttersendingDto, innsendingMottatt: LocalDateTime): Kvittering {
-        val ettersedingRequestData = ettersendingMapper.mapTilIntern(ettersending, innsendingMottatt)
+    fun sendInn(ettersending: EttersendelseDto, innsendingMottatt: LocalDateTime): Kvittering {
+        val ettersedingRequestData: Map<StønadType, EttersendelseDto> = EttersendingMapper.groupByStønad(ettersending, innsendingMottatt)
```

Comment:
Er det noen spesiell grunn til å at mottak skal ha en endepunkt som tar emot Map, mens den returner en liste?
Tenker det kanskje blir litt unødvendig mapping, og att vi i stedet bare kan sende videre listen?

Reply:
Alternativt sendt inn det i en map fra frontend.
Forskjellen mellom liste og map er ju att en liste skulle kunne inneholde flere `EttersendelseDto` med typen stønadstype, i praktiken

Reply:
Det er lettere å lage en god test på dette i api-et enn i frontend for at denne mappingen skal bli "korrekt/bra" - derfor tenker jeg at frontend har en liste - med diverse "valg", men mottak er ikke interessert i en liste - mottak er interessert i "én liste per stønadstype". Da tenker jeg at api er riktig sted å mappe, iallefall.

Reply:
Vet ikke om jeg helt skjønte det med testingen :P 
Man hadde også kunnet løst det med en dto som hadde en liste for hver type, sånn att man enkelt kan gjennbruke det samme DTO'ene rett igjennom :) 
```
data class EttersendelseDto(
val overgangsstønad: List<Dokumentasjonsbehov>,
val skolepenger: List<Dokumentasjonsbehov>,
val barnetilsyn: List<Dokumentasjonsbehov>,
)
```

Reply:
Det objektet kan funke mellom API og Mottak, men ettersending-frontend har ikke noe interesse av å "kategorisere" da du kan legge inn hva som helst og sender inn alt som "en lang liste". Så for frontend sin del vil det objektet bare føre til at mappingen må skje i frontend - og det er jeg ikke så sikker på at er lurt.